graphql/multiplication-connect/createUserData.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from "@apollo/client";
+
+export const CREATE_USER_MC_DATA = gql`
```

Comment:
could you use an upsert here to create if there's not a user in graphql and then add data if it exists?

set up in/then logic to run the query if the user is not null and but then run the upsert if it does not?

here is logic I used: 
const { data } = useQuery<FetchGameLevelResponse>(FETCH_GAME_LEVEL, {
    variables: {
      userId: user.uid,
    },
    onCompleted: (data: FetchGameLevelResponse) => {
      if (data.longestStreakUserData[0] !== undefined) {
        dispatch(initializeGame(data.longestStreakUserData[0].currentLevel));
      } else {
        upsertGameLevel({
          variables: {
            userId: user.uid,
          },
        });
      }
    },
  });

and the upsert looks like this: 
mutation upsertLongestStreakLevel($userId: String = "", $currentLevel: Int = 1) {
    insert_longestStreakUserData_one(object: {userId: $userId, currentLevel: $currentLevel}, on_conflict: {constraint: longestStreakUserData_pkey, update_columns: currentLevel, where: {userId: {_eq: $userId}}}) {
      currentLevel
      
    }
  }